tools/project_manager/update_validate.js

```diff
@@ -0,0 +1,9 @@
+/*global $ validateText */
```

Comment:
Please rename this to something clearer. I would not be able to figure out what this file is and how it is suppose to be used based on the filename and contents alone.

You might just call it `handle_bad_page.js` since it is so closely tied to this page.

Reply:
Done this.